app/assets/javascripts/components/samples.jsx

```diff
@@ -224,10 +224,24 @@ class Samples extends React.Component {
         <div className="sub-header-home">
           <div className="sub-header-items">
             <div className="content">
-
-            <div onClick={ this.gotoPage.bind(this, '/samples/new') }   className="upload">
-                <i className="fa fa-flask" aria-hidden="true"></i>
-                <span>Upload Sample</span>
+              <div className="sub-header-buttons right">
+                <ul>
+                  <li>
+                    <a href='/samples/new'>
+                      <i className="fa fa-flask" aria-hidden="true"></i>
+                      <span>Upload Sample</span> 
+                    </a>
+                  </li>
+                  {(this.project) ? 
+                  <li>
+                    <a href={`/projects/${this.project.id}/visuals`}>
```

Comment:
what does think link show when you click on it? I didn't see the code under app/views/

Reply:
Yeah it takes you to the page, I'm breaking the code into smaller units for easy review :)

Reply:
can you comment on the heat map link for now? you can put it back when it's ready.